V14: Update the login.md article (#6381)
umbraco/UmbracoDocs: 14/umbraco-cms/fundamentals/backoffice/login.md

You will be presented with a login form similar to this:

![Login screen](images/login-backoffice-login-v14.png)

The **login** screen contains a short greeting, a **login form** and an optional **Forgotten password** link.

Below, you will find instructions on how to customize the login screen.

## Greeting

> The login screen features a greeting text: The "Welcome" headline. You can personalize this by overriding the existing language translation keys.

Reviewer: ```suggestion
The login screen features a greeting text: The "Welcome" headline. This can be personalized by overriding the existing language translation keys.
```

> To do this, register a 'localization' manifest for the default language of your Umbraco site, (usually en-US) to override the greetings.

Reviewer: ```suggestion
To do this follow the steps below:
1. Register a 'localization' manifest for the default language of your Umbraco site, (usually en-US) to override the greetings.
```

{% code title="App_Plugins/Login/umbraco-package.json" lineNumbers="true" %}
```json
{
    "alias": "login.extensions",
    "name": "Login extensions",
    "version": "1.0.0",
    "allowPublicAccess": true,
    "extensions": [
        {
            "type": "localization",
            "alias": "Login.Localize.EnUS",
            "name": "English",
            "js": "/App_Plugins/Login/en-us.js",
            "meta": {
                "culture": "en-US"
            }
        }
    ]
}
```
{% endcode %}

> Add an `en-us.js` file containing the following:

Reviewer: ```suggestion
2. Add an `en-us.js` file containing the following:
```

## The Time Out Screen

![Time out screen](./images/timeout-screen.jpg)

The time out screen is displayed when the user has been inactive for a certain amount of time. The screen resembles the login screen in many ways and the two are sometimes confused. The most notable difference is that the time out screen does not have a login form. It only has a message and a button to log in again with Umbraco.

If you have added more than one login provider, the users will also see this screen first. This is because they need to choose which provider to use first. In that case, the screen is also referred to as the **Choose provider screen**.

You can customize the time out screen in the same way as the login screen. The time out screen uses the same localization files as the rest of the Backoffice and **not** those of the login screen. The notable difference is that the time out screen is scoped to the `login` section. The login screen is scoped to the `auth` section of the localization files.

### Greeting

> To update the greeting message on this screen, you will therefore have to change the section to `login`:

Reviewer: ```suggestion
To update the greeting message on this screen, you will have to change the section to `login`:
```
